src/Model/plasticity.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
How's our opinion on the non-standard `#pragma once`?

Reply:
I'll remove it. (I mainly use it for convenience first—and forgot to replace it here... ;)  ... Maybe we'll need some, say, "create-me-an-empty-header-file" script sometime?)